test/tests/ncp-lms/diagonal-ncp-lm-nodal-enforcement.i

```diff
@@ -156,8 +156,8 @@ dt=1
   dt = ${dt}
   dtmin = ${dt}
   solve_type = NEWTON
-  petsc_options_iname = '-snes_max_linear_solve_fail -ksp_max_it -pc_factor_levels'
-  petsc_options_value = '0                           30          16'
+  petsc_options_iname = '-ksp_max_it -pc_factor_levels'
+  petsc_options_value = '30          16'
```

Comment:
```suggestion
  petsc_options_iname = '-snes_max_linear_solve_fail -ksp_max_it -pc_factor_levels -ksp_gmres_restart'
  petsc_options_value = '0                                             60                16                         60'
```

Reply:
Can we try this first? I think in this test I'm attempting to prove that with a stabilization technique applied you can use multigrid preconditioning effectively and that's why I consider a linear solve failure a failure of the test.

Reply:
Clearly my spacing suggestion on github looks awful 😬

Reply:
That's mostly why I asked you about it - it seemed reasonable for it to be there, but just modifying `max_it` really wasn't helping. I also wasn't sure about how far to adjust `pc_factor_levels`. I'll add this manually :D

Reply:
I don't think `-pc_factor_levels` should be playing a role in the AMG test, only in other tests that use this input file. Hopefully modifying the restart will allow the AMG linear solve to converge

Reply:
That works perfectly, and restores the PP output to what was expected before. Thanks for the input!